### PR TITLE
Initializing Dashboard and reworking tests

### PR DESCRIPTION
**Pull request by distortedaxiom**
Adding components and style for the dashboard, displaying the new dashboard elements upon user login.

Reworked the default test cases to work with our new dashboard (homepage)

### CODE REVIEW

File: src/components/pages/Home/HomeContainer.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ function HomeContainer({ LoadingComponent }) {
     return () => (isSubscribed = false);
   }, [memoAuthService]);
 
+  console.log(userInfo);
```
Comment (exploratum) [inline src/components/pages/Home/HomeContainer.js:31]: Console.logs() should not be left  in the code

**Review by exploratum — APPROVED** [review]
Good job - Just a comment that you left a console.log in the code. Please remove it whenever you have a chance

File: src/components/pages/Home/HomeContainer.js (hunk 1 above)
Comment (distortedaxiom) [reply src/components/pages/Home/HomeContainer.js:31]: Will remove for next commit, thank you for the catch!

**Review by karenwinnielei — APPROVED** [review]
Looks great!

**Pull request merged**